boards/arm/stm32f7/stm32f777zit6-meadow/src/stm32_boot.c

```diff
@@ -168,14 +168,15 @@ void board_late_initialize(void)
    * stm32_mpu_uheap((uintptr_t)0x90000000, 0x4000000);
    */
 #endif
+#if defined(CONFIG_MEADOW_HCOM)
+  /* Initialize Meadow HCOM nuttx */
 
-#if defined(CONFIG_NSH_LIBRARY) && !defined(CONFIG_BOARDCTL)
-  /* Perform NSH initialization here instead of from the NSH.  This
-   * alternative NSH initialization is necessary when NSH is ran in
-   * user-space but the initialization function must run in kernel space.
-   */
-
-  board_app_initialize();
```

Comment:
why remove board_app_initialize

Reply:
We don't support custom NuttX apps in our configuration. Meadow.OS provides its own custom userspace. We can bring the define back, and leave it disabled.